### PR TITLE
Adding Russian localization

### PR DESCRIPTION
**Pull request by jonyrh**
Please add Russian localization

### CODE REVIEW

**Comment by euf** [comment]
Me and @imKota are working on the localization, this PR should be rejected

**Comment by imKota** [comment]
I fully agree with @euf

File: iina/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+"general.login" = "Вход в аккаунт";
+"general.logout" = "Выход из аккаунта";
+"general.ok" = "ОК";
+"general.cancel" = "Отмена";
+"general.username" = "Имя пользователя";
+"general.password" = "Пароль";
+
+// PlaylistViewController.swift
+"playlist.playlist_cap" = "ПЛЕЙЛИСТ";
+"playlist.chapter_cap" = "ЗАГОЛОВКИ";
```
Comment (imKota) [inline iina/ru.lproj/Localizable.strings]: ГЛАВЫ будет точнее)

**Comment by jonyrh** [comment]
Whay dublicate? I'm commite Russian localization....

**Comment by saagarjha** [comment]
@euf Why not combine this PR with your in-progress translation? I see no reason to discard @jonyrh's work.

**Comment by euf** [comment]
@saagarjha Unfortunatelly @jonyrh’s work includes translation of only one `.strings` file (there are 25 to be translated) and contains some mistakes. I see no sense in making such a PR, because it will not improve user experience and messes up the translation process. For example, @imKota made 80% ready translation, but instead of PR it he contacted me and we joined our efforts since I also already translated half of the app.

**Comment by jonyrh** [comment]
> and contains some mistakes.
@euf And found many mistakes?

**Comment by imKota** [comment]
@jonyrh Most likely, @euf wanted to say that there are inaccuracies in the translation.

**Comment by jonyrh** [comment]
> wanted to say that there are inaccuracies in the translation.

What exactly is not like? Tell me and I will correct

**Comment by imKota** [comment]
@jonyrh exemple

incorrect translation:
`+"playlist.chapter_cap" = "ЗАГОЛОВКИ";`

correct translation:
`+"playlist.chapter_cap" = "ГЛАВЫ";`

not completely translated:
`+"osd.sub_delay.earlier" = "Subtitle Delay: %.2fs Быстрее";`

incorrect translation:
```
+"osd.abloop.a" = "AB-Петля: A";
+"osd.abloop.b" = "AB-Петля: B";
+"osd.abloop.clear" = "AB-Петля: Убрана
```

correct translation:
```
+"osd.abloop.a" = "AB-повтор: A";
 +"osd.abloop.b" = "AB-повтор: B";
 +"osd.abloop.clear" = "AB-повтор: очищен";
```
incorrect translation:
`+"osd.crop" = "Обрезка: %@";`

correct translation:
`+"osd.crop" = "Кадрирование: %@";
`

incorrect translation:
`+"osd.chapter" = "Заголовок: %@";`

correct translation:
`+"osd.chapter" = "Глава: %@";`

**Comment by imKota** [comment]
@jonyrh присоединяйся в [IINA Telegram group](https://t.me/joinchat/AAAAAEBemW7dU8X7IHShwQ)
Там обсудим.

**Comment by jonyrh** [comment]
@imKota Thank you, I corrected!

**Comment by imKota** [comment]
@jonyrh Look at my PR and compare [#678](https://github.com/lhc70000/iina/pull/678)

**Comment by imKota** [comment]
Close already this pull requests.

**Comment by jonyrh** [comment]
> Look at my PR #678 and compare

@imKota File Localizable.strings is not translated to the end.

**Comment by jonyrh** [comment]
@imKota I just want to help...

**Comment by imKota** [comment]
@jonyrh Yes I know. @euf translated it. Also made changes to my translation.

**Comment by imKota** [comment]
@jonyrh I invite you to the official [IINA group](https://t.me/joinchat/AAAAAEBemW7dU8X7IHShwQ) in the Telegram
I and @euf have already joined this group.

**Comment by jonyrh** [comment]
@imKota No thanks, I do not use Telegram

**Comment by euf** [comment]
Full Russian translation available, please check #686.

**Pull request closed without merging**